ticket_upgrade_prediction/pipeline.py

```diff
@@ -25,7 +25,9 @@ def read_csv_file(self, file_prefix) -> pd.DataFrame:
             self.data_path / f"{file_prefix}_{self.data_type}.csv", sep=";"
         )
 
-    def read_all_files(self) -> (pd.DataFrame, pd.DataFrame, pd.DataFrame):
+    def read_all_files(
+        self,
+    ) -> tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame]:
```

Comment:
which python version are You using? Older ones, doesn't allow typing like that using built in list

Reply:
In that situation it would be best to use:
`from typing import Tuple`
It should work then. 
So i can change it, or You can use newer python version, your call